Kuadrant/multicluster-gateway-controller
# RateLimitPolicy guide to follow after gateway walkthrough

https://github.com/Kuadrant/docs.kuadrant.io/issues/14

Some copied and modified guides as jumping-off points from the multicluster gateway walkthrough, based on:

* https://docs.kuadrant.io/kuadrant-operator/doc/user-guides/simple-rl-for-app-developers/

Main thing changed here is:

- Updated setup, with the view that you've already run the quickstart+gateway walkthrough guide, so you've got Gateways, TLSPolicy+DNSpolicy already setup
-  Rest of the guide is largely as-is, apart from some changes owing to the fact that you'll have working DNS

# Steps to test
- After running quickstart, setup your env to use kuadrant 0.4.0-rc1 catalog source - see https://gist.github.com/jasonmadigan/bde64d4967fdbb740fd8eb876f2579dc. This will pull in RLV2.
- Run through the revised multicluster gateway walkthrough (important, because it created a wildcard gateway listener that we use in the auth + rl guides).
- Step through the RL how-to guide

# Related
- Docs PR for nav updates: https://github.com/Kuadrant/docs.kuadrant.io/pull/29

# Follow-on

Create a similar AuthPolicy guide (parked for now in `14-ap-guide`)
